docs/api/Output.md

```diff
@@ -220,3 +220,24 @@ as subelements with the type of the address part.
 Additional information requested with `extratags=1` and `namedetails=1` can
 be found in extra elements as sub-element of each place.
 
+
+## Notes on field values
+
+### place_id is not a persistent id
+
+The `place_id` is when a Nominatim database gets installed. A single
+place will have a different value on another server or even when the
+same data gets re-imported. It's thus not useful to treat it as
+permanent for later use.
+
+The combination `osm_type`+`osm_id` is slighly better but remember in
```

Comment:
To be precise, the exact unique combination in Nominatim is osm_type+osm_id+class. Should we mention that here?

Reply:
Added in a third paragraph.